bears/c_languages/ArtisticStyleBear.py

```diff
@@ -0,0 +1,865 @@
+    and beautifier for the C, C++, C++/CLI, Objective-C,
+    C# and Java programming languages.
+    """
+
+    LANGUAGES = {'C', 'C++', 'Objective-C', 'C#', 'Java'}
+    REQUIREMENTS = {
+        DistributionRequirement(
+            apt_get='astyle',
+            dnf='astyle'
+        )
+    }
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Formatting'}
+    SEE_MORE = 'http://astyle.sourceforge.net/astyle.html'
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         bracket_style: str='',
+                         use_spaces: bool=None,
+                         indent_size: int=4,
+                         attach_braces_to_namespace: bool=True,
+                         attach_braces_to_class: bool=True,
+                         attach_braces_to_inline: bool=True,
+                         attach_braces_to_extern: bool=False,
+                         allow_indent_classes: bool=True,
+                         allow_indent_modifiers: bool=True,
+                         allow_indent_switches: bool=True,
+                         allow_indent_cases: bool=True,
+                         allow_indent_namespaces: bool=False,
+                         allow_indent_labels: bool=True,
+                         allow_indent_preproc_block: bool=True,
+                         allow_indent_preproc_definition: bool=True,
+                         allow_indent_preproc_conditionals: bool=True,
+                         allow_indent_column_one_comments: bool=True,
+                         allow_pad_header_blocks: bool=True,
+                         allow_pad_operators: bool=True,
+                         allow_pad_parenthesis: bool=False,
+                         allow_pad_parenthesis_out: bool=False,
+                         allow_pad_parenthesis_in: bool=False,
+                         delete_empty_lines_in_func: bool=False,
+                         break_closing_braces: bool=False,
+                         break_elseifs: bool=False,
+                         break_one_line_headers: bool=False,
+                         add_braces_to_one_line_conditionals: bool=False,
+                         remove_braces_from_one_line_conditionals: bool=False,
+                         remove_comment_prefix: bool=True):
+        """
+        :param bracket_style:
+            Brace Style options define the brace style to use.
+            For example: Allman style uses braces that are broken from the
+            previous block. If set to ``allman``, prefer::
+
+                int Foo(bool isBar)
+                {
+                    if (isBar)
+                    {
+                        bar();
+                        return 1;
+                    }
+                    else
+                        return 0;
+                }
+
+            For example: Java style uses braces that are attached to the end of
```

Comment:
I feel we should not enclose words like `Java, Kernighan & Ritchie, Stroustrup, etc.` since those are not code. They just define the styles to use. However, I've enclosed their codewords in backticks in all the examples (If set to `style_name`...).

CC @Makman2 @Asnelchristian